ambry-account/src/main/java/com/github/ambry/account/CompositeAccountService.java

```diff
@@ -0,0 +1,191 @@
+    return primaryAccountService.addAccountUpdateConsumer(accountUpdateConsumer);
+  }
+
+  @Override
+  public boolean removeAccountUpdateConsumer(Consumer<Collection<Account>> accountUpdateConsumer) {
+    return primaryAccountService.removeAccountUpdateConsumer(accountUpdateConsumer);
+  }
+
+  @Override
+  public Collection<Container> updateContainers(String accountName, Collection<Container> containers)
+      throws AccountServiceException {
+    Collection<Container> primaryResult = primaryAccountService.updateContainers(accountName, containers);
+    try {
+      secondaryAccountService.updateContainers(accountName, containers);
+      // Not comparing updated containers list as it could increase latency of update operation.
+    } catch (Exception e) {
+      logger.error("Update containers failed for secondary source", e);
+      accountServiceMetrics.updateContainerInconsistencyCount.inc();
+    }
+    return primaryResult;
+  }
+
+  @Override
+  public Container getContainer(String accountName, String containerName) throws AccountServiceException {
+    Container primaryResult = primaryAccountService.getContainer(accountName, containerName);
+    try {
+      Container secondaryResult = secondaryAccountService.getContainer(accountName, containerName);
+      if (++getContainerRequestCount % 100 < config.samplingPercentageForGetContainerConsistencyCheck
+          && primaryResult != null && !primaryResult.equals(secondaryResult)) {
+        logger.warn("Inconsistency detected between primary and secondary for accountName ={}, containerName = {}",
+            accountName, containerName);
+        accountServiceMetrics.getContainerDataInconsistencyCount.inc();
+      }
+    } catch (Exception e) {
+      accountServiceMetrics.getContainerDataInconsistencyCount.inc();
+      logger.error("get container failed for secondary for accountName={}, containerName={}", accountName,
+          containerName, e);
+    }
+    return primaryResult;
+  }
+
+  @Override
+  public Set<Container> getContainersByStatus(Container.ContainerStatus containerStatus) {
+    Set<Container> primaryResult = primaryAccountService.getContainersByStatus(containerStatus);
+    Set<Container> secondaryResult = secondaryAccountService.getContainersByStatus(containerStatus);
+    if (!primaryResult.equals(secondaryResult)) {
+      logger.warn(
+          "Inconsistency detected between primary and secondary for containers with status ={}, primary ={}, secondary = {}",
+          containerStatus, primaryResult, secondaryResult);
+      accountServiceMetrics.getContainerDataInconsistencyCount.inc();
+    }
+    return primaryResult;
+  }
+
+  @Override
+  public void selectInactiveContainersAndMarkInZK(StatsSnapshot statsSnapshot) {
+    primaryAccountService.selectInactiveContainersAndMarkInZK(statsSnapshot);
+    try {
+      secondaryAccountService.selectInactiveContainersAndMarkInZK(statsSnapshot);
+    } catch (Exception e) {
```

Comment:
`selectInactiveContainersAndMarkInZK` is not implemented in MysqlAccountService.